<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommentDialog</class>
 <widget class="QDialog" name="CommentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>208</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Insert your comments into blockchain</string>
  </property>
  <widget class="QPushButton" name="insertButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Insert into BlockChain</string>
   </property>
   <property name="icon">
    <iconset resource="../feathercoin.qrc">
     <normaloff>:/icons/add</normaloff>:/icons/add</iconset>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>415</y>
     <width>81</width>
     <height>26</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::NoButton</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="addrEdit">
   <property name="geometry">
    <rect>
     <x>82</x>
     <y>25</y>
     <width>451</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Recipient address</string>
   </property>
   <property name="statusTip">
    <string>Recipient address</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>29</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Pay to:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>75</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Comments</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>140</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16177186</height>
    </size>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="icon">
    <iconset resource="../feathercoin.qrc">
     <normaloff>:/icons/quit</normaloff>:/icons/quit</iconset>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtComment">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>451</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../feathercoin.qrc"/>
 </resources>
 <connections/>
</ui>
